osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -214,6 +205,8 @@ protected override void LoadComplete()
         protected override void Update()
         {
             base.Update();
+
+            topRightElements.Y = ToLocalSpace(ScoreCounter.Drawable.ScreenSpaceDrawQuad.BottomRight).Y;
```

Comment:
I'm not a huge fan of this - it's now being done in two places, but let's go with it for now.

Reply:
wait until you see the next PR! but yeah, this will be replaced over time with something more appeasing to the eyes.